api-docs/leads.md

```diff
@@ -0,0 +1,332 @@
+# Leads
+
+The leads document direct encounters with leads.
+
+Leads are potiental clients.
+
+Here is the list of the valid lead types:
+
+- `leadCreated`,
+- `leadUpdated`,
+- `leadDeleted`
+
+## Get a single Lead
+
+### Request
+
+```http
+GET /api/leads/:id
+```
+
+### Response
+
+```json
+{
+  "id": 1,
+  "dateOfSignup": "2019-09-17",
+  "leadStatus": "pending",
+  "firstName": "Joel",
+  "lastName": "Denning",
+  "fullName": "Joel Denning",
+  "phone": "555-555-5555",
+  "smsConsent": true,
+  "zip": "84115",
+  "age": 25,
+  "gender": "male",
```

Comment:
Gender is not asked for in https://docs.google.com/spreadsheets/d/1wGx9pu2Udnn1MEWhas3u-PTe8uwZeStj20ItJ0UDirA/edit#gid=0

I think we should only expect leads to have the information that is already in that form.